README.md

```diff
@@ -1,18 +1,680 @@
+
+But we recommend that you use the Vale extension for your IDE and follow the suggestions.
+
+If you have VSCode, we recommend the Vale VSCode extension.
+
+For details on the styles, see the rulesets defined in the `vale/styles` configuration.
+
+#### Capitalization of core terms
+
+Many of Temporal's core terms can be used in a generic way.
+
+To differentiate one of Temporal's core terms from a generic instance of a term, always treat the Temporal term as a proper noun in documentation.
+
+Generic versions of the same term should not be capitalized and should be used sparingly to avoid confusion.
+
+- Correct: "Next, register the Activity within the Workflow."
+
+- Incorrect: "Next, register the activity within the workflow."
+
+#### Abbreviation of "identifier"
+
+In text, do not abbreviate the word "identifier" as "ID", "Id", or "id" unless it is part of a Temporal core term, such as "Workflow Id" or "Activity Id".
+
+- Correct: "You can provide an order identifier or customer identifier as a Workflow Id."
+
+- Incorrect: "You can provide an order ID or customer id as a Workflow Id."
+
+In code (and when quoting or referring to code in text), follow the conventions of each language.
+
+#### En dashes in ranges
+
+Using an en dash (`&ndash;` or the character `–`) in a range of numbers is acceptable.
+Even better is to use words such as _from_, _to_, and _through_.
+
+#### Be consistent.
+
+If you use an en dash in one range, use en dashes in all ranges.
+Do not mix words and en dashes (or hyphens, for that matter).
+
+- Correct: "5 to 10 GB"
+- Correct: "5–10 GB"
+- Correct: "5-10 GB"
+- Incorrect: "from 5-10 GB"
+
+#### Infinitive verb forms in headings
+
+Titles and headings should use infinitive verb forms whenever possible. People tend to search by using infinitive verb forms, so using them helps SEO.
+
+- Correct: "How to install Temporal"
+- Incorrect: "Installing Temporal"
+
+#### Infinitive verb forms in labels
+
+- Correct: "Install Temporal"
+- Incorrect: "Installing Temporal"
+
+#### Sentence casing in headings
+
+Use sentence casing for titles and headings.
+Sentence casing means that only the first letter of the first word and proper nouns are capitalized.
+
+- Correct: "How to get started with Temporal"
+- Incorrect: "How To Get Started With Temporal"
+
+### What is the philosophy around versioning the documentation?
+
+The Temporal Platform includes many different components and core dependencies. Each of these components is often independently versioned and documents their stability and support for their own dependencies. For example, the [Temporal Go SDK reference](https://pkg.go.dev/go.temporal.io/sdk?tab=versions) provides that view.
+
+The goal of this information set, in regards to versioning, is to remain “current”. That is, this information should serve the needs of the Platform’s user base as best it can based on what is/has recently happened across all the components of the Temporal Platform.
```

Comment:
```suggestion
The goal of versioning is to remain “current”. In other words, the Temporal information corpus should serve the needs of the user base as best it can based on recent changes across the Temporal Platform.
```

Reply:
This changes the meaning.
The goal of the information set is not the same as the goal of versioning.